pkg/vz/errors.go

```diff
@@ -0,0 +1,8 @@
+//go:build darwin
+// +build darwin
```

Comment:
Do we need this tags ?? 
errors.go can be generic irrespective of platform (We can move this https://github.com/lima-vm/lima/blob/master/pkg/vz/vz_driver_others.go#L13 as well here)

If we need this tag, then naming the file as errors_darwin.go should be sufficient and consistent across i believe

Reply:
Without build tags the variable is detected as unused in non-darwin builds.

Reply:
Renamed the file, and I updated build tags as more were required due to your previous comment: https://github.com/lima-vm/lima/pull/1155#discussion_r1029696381